src/components/20-molecules/datepicker/ui.test.js

```diff
@@ -103,14 +103,13 @@ test('should begin with September 1 as first Monday in 1980', async t => {
   await t.expect(firstVisibleDay.innerText).eql('1');
 });
 
-// Preselected is: 22. Mai 2020
-test('should have working selection and preselection', async t => {
+test('should have working selection', async t => {
```

Comment:
Is that a test for the current day to be "selected"? If not, please add a test to check if the current day is visually highlighted.